src/test/resources/features/EditGame.feature

```diff
@@ -73,3 +73,31 @@ Feature: Edit Game
     Then The response code is 404
     And It has not been edited a game with id "1"
 
+  Scenario: Edit game to change the starting time
+    Given I login as "admin" with password "password"
+    And I register a new game with id "1"
+    When I edit game with id "1"
```

Comment:
This "I edit game with id..." step that just uses an empty patch is not required in this case. Keep just the step you are testing in all 3 cases, for instance, "When I edit the game with id "1" and set the start date..."